docs/Node Operators/configuration-variables.md

```diff
@@ -810,3 +884,10 @@ The default timeout for HTTP requests.
 `10ms`
 `1h15m`
 `42m30s`
+
+> ⚠️ NOTE
+> Some env vars require a file size. A file size string is an unsigned integer (123) or float (12.3) followed by a unit suffix. Valid file size units are "b", "kb", "mb", "gb", and "tb". If the unit is omitted, it is assumed to be "b" (bytes). Capitalization does not matter. Some examples:
+
+`123gb`
+`1.2TB`
```

Comment:
Let's clarify if we mean base-10 or base-2.  For the larger size units, this makes a significant difference.